src/constants.ts

```diff
@@ -117,7 +117,7 @@ export const CONDUIT_KEYS_TO_CONDUIT = {
   [CROSS_CHAIN_DEFAULT_CONDUIT_KEY]: CROSS_CHAIN_DEFAULT_CONDUIT,
 };
 
-export const DEFAULT_ZONE = "0x9b814233894cd227f561b78cc65891aa55c62ad2";
+export const DEFAULT_ZONE = "0x004c00500000ad104d7dbd00e3ae0a5c00560c00";
```

Comment:
This zone is only applicable for Ethereum, and it changes based on if it's an english auction or not

English auction uses old zone 0x9b814233894cd227f561b78cc65891aa55c62ad2, all other orders use new zone only for ethereum.

Reply:
We don't support posting english auctions through the UI right now. Does that mean rinkeby should use old zone?

Reply:
Rinkeby should just use the old zone for now.

Reply:
yeah I mean we don't support posting english auctions through the new orders endpoints

Reply:
Hm, okay well then yeah let's just change it based on network then